spec/support/corpus.rb

```diff
@@ -80,21 +82,11 @@ def verify_mutation_generation
           start:        method(:start),
           in_processes: parallel_processes
         }
-        total = Parallel.map(effective_ruby_paths, options) do |path|
-          count = 0
-          node =
-            begin
-              Parser::CurrentRuby.parse(path.read)
-            rescue EncodingError, ArgumentError
-              nil # Make rubocop happy
-            end
-
-          if node
-            count += Mutant::Mutator::REGISTRY.call(node).length
-          end
 
-          count
+        total = Parallel.map(effective_ruby_paths, options) do |path|
+          filter_exceptions(path, DEFAULT_MUTATION_COUNT) { count_mutations(path) }
```

Comment:
As this is the only call side for `filter_exceptions` passing in the 2nd argument from a constant is redundant.